Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/CompositeSequences/EconomizerMultiZone.mo

```diff
@@ -0,0 +1,185 @@
+within Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.CompositeSequences;
+model EconomizerMultiZone "Multiple zone VAV AHU economizer control sequence"
+
+  parameter Boolean use_enthalpy = true
+    "Set to true if enthalpy measurement is used in addition to temperature measurement";
+  parameter Real kPIDamLim=1 "Gain of damper limit controller";
+  parameter Real kPIMod=1 "Gain of modulation controller";
+  parameter Modelica.SIunits.Time TiPIDamLim=0.9 "Time constant of damper limit controller integrator block";
+  parameter Modelica.SIunits.Time TiPIMod=300 "Time constant of modulation controller integrator block";
+
+  CDL.Interfaces.RealInput TCooSet(unit="K", quantity = "ThermodynamicTemperature")
+    "Supply air temperature cooling setpoint" annotation (Placement(transformation(
+    extent={{-140,30},{-120,50}}), iconTransformation(extent={{-120,10},{-100,30}})));
+  CDL.Interfaces.RealInput TSup(unit="K", quantity = "ThermodynamicTemperature")
+    "Measured supply air temperature" annotation (Placement(transformation(
+    extent={{-140,50},{-120,70}}), iconTransformation(extent={{-120,30},{-100,50}})));
+  CDL.Interfaces.RealInput TOut(unit="K", quantity = "ThermodynamicTemperature")
+    "Outdoor air (OA) temperature" annotation (Placement(transformation(extent={{-140,130},{-120,150}}),
+    iconTransformation(extent={{-120,110},{-100,130}})));
+  CDL.Interfaces.RealInput TOutCut(unit="K", quantity = "ThermodynamicTemperature")
+    "OA temperature high limit cutoff. For differential dry bulb temeprature condition use return air temperature measurement"
+    annotation (Placement(transformation(extent={{-140,110},{-120,130}}),
+        iconTransformation(extent={{-120,90},{-100,110}})));
+  CDL.Interfaces.RealInput hOut(unit="J/kg", quantity="SpecificEnergy") if use_enthalpy
+    "Outdoor air enthalpy" annotation (Placement(transformation(extent={{-140,90},{-120,110}}),
+    iconTransformation(extent={{-120,70},{-100,90}})));
+  CDL.Interfaces.RealInput hOutCut(unit="J/kg", quantity="SpecificEnergy") if use_enthalpy
+    "OA enthalpy high limit cutoff. For differential enthalpy use return air enthalpy measurement"
+    annotation (Placement(transformation(extent={{-140,70},{-120,90}}),
+        iconTransformation(extent={{-120,50},{-100,70}})));
+  CDL.Interfaces.RealInput VOut_flow(unit="m3/s")
+    "Measured outdoor volumetirc airflow rate [fixme: which quantity attribute should we use? add for all V]"
```

Comment:
Address all fixme for a pull request.
Volumetric has a typo.
For quantity, see Modelica.SIunits.VolumeFlowRate